### DISCUSSION TITLE
Issue with the way recordings are saved

### DISCUSSION THREAD

**Discussion by DasDunkel** [blakeblackshear/frigate#14724] [General Support]
Currently as I understand it, frigate is capturing a timelapse instead of storing all of the real-time footage, this is great for saving space when there's nothing interesting happening, love this, but it can cause issues with events/alerts, more specifically important frames being discarded.

What I'd like to be able to do is have all events and a given time around them be stored as-is with no extra filtering or encoding, exactly what the camera shows.  (Of course excluding encoding to a suitable codec)

It's possible that I'm missing something, but currently all non-live footage is played back at an increased speed (I've checked playback speed, and default playback speed)

**Comment by NickM-27** [comment]
> Currently as I understand it, frigate is capturing a timelapse instead of storing all of the real-time footage, this is great for saving space when there's nothing interesting happening, love this, but it can cause issues with events/alerts, more specifically important frames being discarded.

this is incorrect, there is no "instead of". A preview is saved for any hour that real time recordings are saved.

**Reply by DasDunkel** [reply]
If frigate does indeed store real-time recordings, I'll have to assume that there's something wrong with my install/configuration, as even when viewing the files directly they are clearly sped up by about 2x making them look like a timelapse (My camera adds a timestamp in the corner which confirms this)

I'm pretty sure it isn't the hardware being overloaded, CPU and GPU are always under 50% usage, and there's nothing in the logs that would indicate an issue with keeping up

Are there any pointers you could give on things I could try?

**Reply by NickM-27** [reply]
you haven't provided any information on config so there are no suggestions to really make, Frigate copies recordings directly from the camera so this is not something that Frigate is doing

**Reply by DasDunkel** [reply]
Config here: https://txt.dunk.dev/dunk/0531d06f70eb4665aa24916c38197b1a

The part I'm struggling to understand is, if it does copy exactly like you say, why does the live view give a perfectly smooth real time view, but **all** recordings play as a choppy timelapse regardless of the thing used to play them

**Reply by NickM-27** [reply]
Why are you reencoding the stream? That might be part of the issue

Live and recording use different players

**Reply by DasDunkel** [reply]
Which part are you talking about?
If you mean the stuff in go2rtc, because it's mjpeg and the docs recommended doing this

**Reply by NickM-27** [reply]
You'll need to set the frame rate accordingly then, perhaps better is a mismatch there

**Reply by DasDunkel** [reply]
How would I go about doing that? I can't seem to find anything in the docs for frigate or go2rtc

**Reply by NickM-27** [reply]
Ah yeah that's only for usb device. Not sure why this would be occurring but it seems to be specific to that camera as we've not had other reports of the issue.
